src/controller_examples/rabbitmq_controller/proof/liveness/liveness.rs

```diff
@@ -163,35 +208,96 @@ proof fn invariants_is_stable(rabbitmq: RabbitmqClusterView)
     );
 }
 
+spec fn invariants_phase_II(rabbitmq: RabbitmqClusterView) -> TempPred<RMQCluster> {
```

Comment:
Could you document phase 2, 3, 4, 5, and 6? Just document intuitively what are these phases and why we want to split the entire proof process into such phases.

Reply:
yes will do

Reply:
Added some comments.